README.md

```diff
@@ -1,67 +1,93 @@
+
+This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for Conjur.
 
 [![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
 
 ---
 
-### **Status**: Alpha
+## Certificate level
 
-#### **Warning: Naming and APIs are still subject to breaking changes!**
+### Conjur CLI
 
----
+![](https://img.shields.io/badge/Certification%20Level-Certified-6C757D?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+
+The Conjur CLI is a **Certified** level project. It's been reviewed by CyberArk to verify that it will securely
+work with CyberArk DAP as documented. In addition, CyberArk offers Enterprise-level support for these features. For
+more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
+
+### Conjur Python SDK
 
-## Installing the code
+![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+
+The Conjur Python SDK is a **Community** level project. It's a community contributed project that **is not reviewed or supported
+by CyberArk**. For more detailed information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).
 
 ### Using conjur-api-python3 with Conjur OSS 
 
 Are you using this project with [Conjur OSS](https://github.com/cyberark/conjur)? Then we 
 **strongly** recommend choosing the version of this project to use from the latest [Conjur OSS 
 suite release](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html). 
 Conjur maintainers perform additional testing on the suite release versions to ensure 
-compatibility. When possible, upgrade your Conjur version to match the 
-[latest suite release](https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm); 
-when using integrations, choose the latest suite release that matches your Conjur version. For any 
+compatibility. When possible, upgrade your Conjur OSS version to match the 
+[latest suite release](https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm).
+When using integrations, choose the latest suite release that matches your Conjur OSS version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+### Supported Services
+
+- Conjur OSS v1.2.0 or later
```

Comment:
@sigalsax  Didn't we write in the docs v1.5.0 or later?

Reply:
should be 1.2